plugin/src/main/scala-3/chisel3/internal/plugin/BundlePhase.scala

```diff
@@ -86,7 +86,9 @@ object BundleHelpers {
       case _ =>
     })
 
-    Some(constructor.termParamss.map(_.map { case vp =>
+    def isImplicit(x: tpd.ValDef): Boolean = x.symbol.flags.is(Flags.Implicit)
+
+    Some(constructor.termParamss.map(_.filterNot(isImplicit(_)).map { case vp =>
```

Comment:
Parameter lists should either all be implicits or all not be implicits so you should be able to filter the entire list by just the first argument